rsyslog/init.sls

```diff
@@ -50,6 +51,33 @@ rsyslog_mmjsonparse:
     - require:
       - pkg: rsyslog
 
+{%- else %}
+rsyslog_updates:
+  pkgrepo.managed:
+    - name: deb http://ppa.launchpad.net/adiscon/v8-devel/ubuntu focal main
+    - dist: focal
+
+rsyslog:
+  pkg.installed:
+    - name: rsyslog
+    - version: 8.2008.0~20200811150537-0adiscon1focal
+    - require:
+      - remove_rsyslog_8_16_0
```

Comment:
not needed anymore

Reply:
I wasn't sure about these since some HLCs _might_ have the old version still.